PKHeX.Core.AutoMod/Teams/SmogonSetList.cs

```diff
@@ -58,12 +58,28 @@ private void LoadSetsFromPage()
             for (int i = 1; i < split1.Length; i++)
             {
                 var shiny = split1[i - 1].Contains("\"shiny\":true");
+                var level = split1[i - 1].Contains("\"format\":\"LC") ? 5 : 100;
+                if (!split1[i - 1].Contains("\"level\":0,") && split1[i - 1].Contains("\"level\":"))
+                {
+                    int.TryParse(split1[i - 1].Split(new[] {"\"level\":"}, StringSplitOptions.None)[1]
+                        .Split(',')[0], out level);
+                }
+
+                var gender = 'D';
+
+                // Gender check; seemingly unused as alternate gender forms are listed as separate entries
+                //if (!split1[i - 1].Contains("\"gender\":\"DC\"") && split1[i - 1].Contains("\"gender\":"))
+                //{
+                //    gender = split1[i - 1].ToCharArray()[
+                //        split1[i - 1].IndexOf("\"gender\":\"") + "\"gender\":\"".Length];
+                //}
+
```

Comment:
Lets just remove the gender check for now since we dont have gender specific pokemon and ALM handles single gendered mons anyway!